src/main/webapp/Register.jsp

```diff
@@ -0,0 +1,152 @@
+<!DOCTYPE html>
+<html>
+<head>
+<meta name="viewport" content="width=device-width, initial-scale=1">
+<script>
+function validate()
+{
+     var firstName = document.form.firstName.value;
+     var lastName = document.form.lastName.value;
+     var city = document.form.city.value;
+     var state = document.form.state.value;
+     var country = document.form.country.value;
+     var zip =document.form.zip.value;
+     var phone = document.form.phone.value;
+     var password = document.form.password.value;
+
+     if (firstName==null || firstName=="")
+     {
+     alert(" FirstName can't be blank");
+     return false;
+     }
+     else if (lastName==null || lastName=="")
+     {
+     alert("LastName can't be blank");
+     return false;
+     }
+     else if(password.length<6)
+     {
+     alert("Password must be at least 6 characters long.");
+     return false;
+     }
+ }
+</script>
+<style>
+body {
+  font-family: Arial, Helvetica, sans-serif;
```

Comment:
Please don't use CSS styling in the JSP file. I asked you guys to put CSS content in an external CSS file. Please use the right practice to do so.